README.adoc

```diff
@@ -21,50 +21,50 @@ permalink: /guides/jenkins-integration-with-kabanero/
+In this guide, you will learn how to build an image and how to use Jenkins to deploy an application that uses that image. The link:https://github.com/kabanero-io/kabanero-pipelines[kabanero-pipelines] repository contains a collection of tasks and pipelines that are intended to work out of the box with application stacks to illustrate a CI/CD workflow.
+The pipelines and tasks can be run manually or by using a webhook. This guide shows you how to use an automated webhook to create and run pipelines.
 
-The following diagram depicts the relationship and flow between Tekton and Jenkins:
+The following diagram depicts the relationship and flow between the pipeline and Jenkins:
 
-image::/img/guide/jenkins-tekton.png[link="/img/guide/jenkins-tekton.png" alt="This diagram depicts the relationship and flow between Tekton and Jenkins."]
+image::/img/guide/jenkins-tekton.png[link="/img/guide/jenkins-tekton.png" alt="This diagram depicts the relationship and flow between the pipeline and Jenkins."]
 
 == Intended audience
-The purpose of the guide is to show an existing Jenkins user or administrator how to build an image with Tekton and how to deploy an image with Jenkins. Users should have experience with Jenkins and be familiar with the fundamental concepts of microservices, Docker, Kubernetes, and Tekton.
+The purpose of the guide is to show an existing Jenkins user or administrator how to build an image with the pipeline and how to deploy an image with Jenkins. Users should have experience with Jenkins and be familiar with the fundamental concepts of microservices, Docker, Kubernetes, and pipelines.
 
 == Prerequisites
 You need the following prerequisites to complete the guide:
 
 * link:https://www.openshift.com/products/container-platform[OpenShift 4.2+] or link:https://www.okd.io/[OKD 3.11+]
 * link:https://github.com/Kabanero-io/Kabanero-foundation[Kabanero Foundation] (cloned and installed)
-// Kabanero Pipelines
-* link:https://github.com/Kabanero-io/Kabanero-pipelines[Kabanero Pipelines] (cloned)
+// Pipelines
+* link:https://github.com/Kabanero-io/Kabanero-pipelines[Pipelines] (cloned)
 // Appsody Installation
 * link:http://appsody.dev[Appsody] (installed)
 
 See each project's documentation for necessary system requirements.
 
-== Build an image with Tekton using an automated webhook
+== Build an image with an automated webhook
 
-. Log in as cluster-admin in the master node of your OKD cluster. The following steps assume that you are logged in as cluster-admin, and the commands are run on the master node of your OKD cluster.
-. While logged in as a cluster-admin, run the installation script from the link:https://github.com/kabanero-io/kabanero-foundation[kabanero-operator release]:
+. Log in as cluster-admin in the master node of your Kubernetes cluster. The following steps assume that you are logged in as cluster-admin, and the commands are run on the master node of your Kubernetes cluster.
```

Comment:
I'm curious why this guide has steps to install kabanero - surely we assumed they had installed kabanero already?  I actually think it is more dangerous to describe install here because it will become stale and is not visible to folks who maintain kabanero install.